NewProjectCI/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.7.10"
+    application
+}
+
+group = "org.example"
+version = "1.0-SNAPSHOT"
+
+application {
+    mainClass.set("Main.kt")
+}
+
+repositories {
+    mavenCentral()
+}
+// https://stackoverflow.com/questions/26469365/building-a-self-executable-jar-with-gradle-and-kotlin
```

Comment:
It's not clear what this comment refers to.

Reply:
oh this was a debug issue that I solve